terraform-aws-zhe-backend/variables.tf

```diff
@@ -0,0 +1,203 @@
+variable "region" {
+  description = "AWS region"
+  type        = string
+}
+
+variable "env" {
+  description = "Tag to identify zhe enviroment type and resources"
+  type        = string
+  default     = ""
+}
+
+variable "creator" {
+  description = "Tag to identiy who created this module's resources"
+  type        = string
+  default     = "Terraform"
+}
+
+variable "bucket_force_destroy" {
+  description = "A boolean that indicates all objects (including any locked objects) should be deleted from the bucket so that the bucket can be destroyed without error. These objects are not recoverable."
```

Comment:
```suggestion
  description = "A boolean that indicates all objects (including any locked objects) that should be deleted from the bucket so that the bucket can be destroyed without error. These objects are not recoverable."
```